src/modules/rover_pos_control/RoverPositionControl.hpp

```diff
@@ -127,14 +129,25 @@ class RoverPositionControl final : public ModuleBase<RoverPositionControl>, publ
 	uORB::Subscription _vehicle_angular_acceleration_sub{ORB_ID(vehicle_angular_acceleration)};
 	manual_control_setpoint_s		_manual_control_setpoint{};			    /**< r/c channel data */
 	position_setpoint_triplet_s		_pos_sp_triplet{};		/**< triplet of mission items */
-	vehicle_attitude_setpoint_s		_att_sp{};			/**< attitude setpoint > */
-	vehicle_rates_setpoint_s		_rates_sp{};			/**< rate setpoint > */
+
+	// [rad] Attitude setpoint in global frame (North - East - Down)
+	vehicle_attitude_setpoint_s		_att_sp{};
+
+	// [rad/s] Angular rates setpoint in local frame (Front - Right - Down)
+	vehicle_rates_setpoint_s		_rates_sp{};
```

Comment:
I cleared up the unit specification (and the reference frame), as it was getting quite confusing (since we use global / local frame & radians / degrees) in the code.